packages/app/features/top/ChannelScreen.tsx

```diff
@@ -64,11 +64,6 @@ export default function ChannelScreen(props: Props) {
   const channelIsPending = !channel || channel.isPendingChannel;
   useFocusEffect(
     useCallback(() => {
-      if (!channelIsPending) {
```

Comment:
was this intentionally changed?

Reply:
Yes, I found that keeping this in the useFocusEffect caused unnecessary syncs that would then trigger unnecessary react query invalidations through the change listener.